third_party/ibis/ibis_addon/api.py

```diff
@@ -67,10 +74,13 @@ def same_type(from_type, to_type) -> bool:
     return op.to_expr()
 
 
-def force_cast(self, target_type: dt.DataType) -> Value:
+def force_cast(self, target_type: str) -> Value:
```

Comment:
Wondering if changing data type for `target_type` argument didn't cause any side effects on other parts this method is used?

Reply:
I think we should cast it into the standard form which includes hyphens - humans are going to read it and we want it to be understandable.

Reply:
Regarding changing "dt.DataType" to "str". That is just a type hint and I did it because when debugging my changes I noticed that we pass a string data type in to this function and not an Ibis data type. I was just correcting the hint to match what is being done in the code.

Reply:
> Regarding changing "dt.DataType" to "str". That is just a type hint and I did it because when debugging my changes I noticed that we pass a string data type in to this function and not an Ibis data type. I was just correcting the hint to match what is being done in the code.

Ahh okay, no worries! Forgot about it being just a type hint and not an enforcement, all good!